android/clientlib-ktx/src/main/java/com/solana/mobilewalletadapter/clientlib/scenario/ScenarioChooser.kt

```diff
@@ -0,0 +1,14 @@
+package com.solana.mobilewalletadapter.clientlib.scenario
+
+class ScenarioChooser {
```

Comment:
The idea here is to have a class that can find and create the right scenario class based on the given type as other options are coming in the future. Thoguhts?

Reply:
There are pretty different actions that need to be taken depending on the chosen association method. For example, displaying a QR code, sending an Intent, etc. It seems that choosing the scenario is already a fairly involved operation, and this helper class may not actually end up abstracting away enough of the details to be useful? Or would the intention be to have this class act as a pretty complex factory that can produce everything that is required by a given Scenario?

Reply:
I was wondering about this. It's probably too much to ask to know ahead of time what we're going to need from this class. This class already isn't that great, so it's just going to get in our way. I'll remove for now.